packages/@vue/cli/lib/util/ProjectPackageManager.js

```diff
@@ -76,7 +77,8 @@ class PackageManager {
     }
 
     if (!SUPPORTED_PACKAGE_MANAGERS.includes(this.bin)) {
-      throw new Error(`Unknown package manager: ${this.bin}`)
+      warn(`Unknown package manager: ${this.bin}`)
```

Comment:
Looks good to me but we need a stronger tone here.
Need to convey the message that the unknown package manager may not be well supported and we are now treating it like `npm` but potential errors could happen.